Specified the locale in setupIntl()

ijlee2 opened the pull request:
In `ember-intl@v7`, the test helper `setupIntl()` [requires end-developers to specify a locale](https://ember-intl.github.io/ember-intl/versions/v7.0.2/docs/migration/v7#required-locale-in-test-helpers). I updated the blueprint file for `tests/helpers/index.ts` to reflect this breaking change.

> [!NOTE]
>
> The change to `tests/helpers/index.ts` is backward-compatible, because specifying the locale is optional in earlier versions of `ember-intl`.

Two possible alternatives:

1. Remove the commented line from `setupApplicationTest()`, because the test helper `setupIntl()` is really meant to help end-developers write rendering and unit tests.
1. Move the commented line to `setupRenderingTest()`, in case an end-developer wants an example of calling a function (e.g. `setupIntl()`) in all or most rendering tests.

ef4 commented:
I think the test failures here are caused by an ember-data beta release problem that has been fixed, not sure if it's been released.

This PR seems fine and should merge whenever we can get CI green again. We do need to confirm that no tests are failing just because of the blueprint changes.

kategengler commented:
@ijlee2 Can you try rebasing to see if that fixes CI?

ijlee2 commented:
@kategengler Thanks for letting me know. The branch had already been rebased, after which CI had failed, so I cherry-picked the commit to re-trigger CI just now.

kategengler commented:
I'm happy to merge now but do you want to retarget to beta so it makes it into 5.10?

ijlee2 commented:
I rebased the branch against `v5.10.0-beta.0` and changed the target branch to `beta`. I hope it's right; if not, feel free to change the pull request, as I'll be out for several days. Thanks.
